application/common/core.cpp

```diff
@@ -43,19 +44,61 @@ void loop()
 
     // Get the stage as it will receive most events.
     Loom2D::Stage *stage = Loom2D::Stage::smMainStage;
-    
+
+    if(gPendingResizeEvent && stage)
```

Comment:
Does this make some of the initialization logic in Stage ctor and/or Stage::updateFromConfig redundant?